src/test/java/hr/fer/axon/TestContainersFixture.java

```diff
@@ -18,52 +18,62 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
 import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
 import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.ContextConfiguration;
-
-import jakarta.annotation.PostConstruct;
+import org.testcontainers.containers.PostgreSQLContainer;
 
 @Tag("integration")
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, properties = {
-    "logging.level.org.springframework.security=DEBUG"
+  "logging.level.org.springframework.security=DEBUG"
 })
-@ContextConfiguration(initializers = {DbContainerInitializer.class})
 @ActiveProfiles({"test"})
 public abstract class TestContainersFixture {
   private static final Logger log = LoggerFactory.getLogger(TestContainersFixture.class);
 
+  @ServiceConnection
```

Comment:
this is just a FYI how "simply" we can configure the TC database nowadays, nothing wrong with your old approach

Reply:
Yes that is much simpler